### ISSUE TITLE
Studio: make the generated selector configurable

### ISSUE THREAD

**Issue by iam-jam** [cypress-io/cypress#15467]
### What would you like?
To provide configuration to Studio so I can control the generated selector. 

```json
{
   "experimentalStudio":  {
      "preferSelector": "[data-test-id]"
   }
}
```

### Why is this needed?
When using studio some of my generated tests look like:

```js
cy.get('#el-21ddf848e73d > .app > .btn').click();
cy.get('#el-342d847f60d5').click();
```

In my opinion, using IDs or paths has the potential to be flakey and so it would be much better if we could get Studio to use the data attribute as we would do when writing tests manually.

**Comment by iam-jam** [comment]
Seems like this would be solved if #2728 was implemented.

**Comment by iam-jam** [comment]
OK I missed https://docs.cypress.io/api/cypress-api/selector-playground-api.html#Arguments

Changing my attribute to `[data-testid]` solves the issue :)
